### PR TITLE
add Staging environment and cleanup

### PR DESCRIPTION
**Pull request by JasonTheMain**
## Changes proposed in this pull request:
- Adding staging environment as option
- updating development to support not being a solo deploy anymore
-

## Security considerations

None

### CODE REVIEW

File: opsfiles/enable-proxy-auth-staging.yml
Comment (markdboyd) [inline opsfiles/enable-proxy-auth-staging.yml:1]: Is there is any difference between this and `enable-proxy-auth-dev.yml`?

Comment (JasonTheMain) [reply opsfiles/enable-proxy-auth-staging.yml:1]: Line 98

Comment (JasonTheMain) [reply opsfiles/enable-proxy-auth-staging.yml:1]: We could make that one orphaned to a different file

Comment (JasonTheMain) [reply opsfiles/enable-proxy-auth-staging.yml:1]: using credhub we could make it one file

Comment (markdboyd) [reply opsfiles/enable-proxy-auth-staging.yml:1]: OK, whatever you think will work best

**Review by markdboyd — APPROVED** [review]
Great work!

**Pull request merged**